Utils.lua

```diff
@@ -701,6 +688,7 @@ end
 function UTILS.LibStClassCellUpdate(rowFrame, frame, data, cols, row, realrow, column, fShow, table, ...)
     local class = data[realrow].cols[column].value
     if class then
+        class = supper(sgsub(class, "%s", ""))  -- remove space and to uppercase
```

Comment:
if we could actually use numbers here we could do simple classnumber to class mapping here